IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDWebviewUIController.m

```diff
@@ -52,6 +54,13 @@ + (void)initialize
     });
 }
 
++ (WKWebViewConfiguration *)createWebViewConfigWithPKeyAuthUserAgent
+{
+    WKWebViewConfiguration *webConfig = [WKWebViewConfiguration new];
+    webConfig.applicationNameForUserAgent = kMSIDPKeyAuthKeyWordForUserAgent;
```

Comment:
I don't think we support PkeyAuth on macOS for ADAL. Let's remove it from this version.